src/testing.rs

```diff
@@ -198,11 +199,19 @@ pub mod environment {
 	}
 
 	impl crate::voter::Environment<&'static str, u32> for Environment {
-		type Timer = BoxFuture<'static, Result<(),Error>>;
+		type Timer = Pin<Box<dyn Future<Output = Result<(), Error>> + Unpin + Send + Sync>>;
```

Comment:
I need to double check why we need to pin this and the stream/sink below.

Reply:
Removed in latest